be/src/util/json_converter.cpp

```diff
@@ -37,8 +38,16 @@ class SimdJsonConverter {
             return JsonValue(builder.slice());
         } catch (simdjson::simdjson_error& e) {
             std::string_view view(value.get_raw_json_string().raw());
-            auto err_msg = strings::Substitute("Failed to convert simdjson value, json=$0, error=$1", view.data(),
-                                               simdjson::error_message(e.error()));
+            // truncate the raw json string if it is too large
```

Comment:
it should be much easier to cover this by an unit test than a sql tester.

Reply:
updated